### PR TITLE
migrate README from jQuery

### PR DESCRIPTION
**Pull request by amilajack**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,9 +53,9 @@ functions. This allows you to control *what* to highlight and *when*.
 Here’s an equivalent way to calling [`initHighlightingOnLoad`][1] using
 jQuery:
 
-```javascript
-$(document).ready(() => {
-  $('pre code').each((i, block) => {
+```js
+document.addEventListener('DOMContentLoaded', (event) => {
+  document.querySelectorAll('pre code').forEach((block, i) => {
```
Comment (marcoscaceres) [inline README.md]:
```suggestion
  document.querySelectorAll('pre code').forEach((block) => {
```

File: README.md [hunk 2]
```diff
@@ -65,10 +65,10 @@ You can use any tags instead of `<pre><code>` to mark up your code. If
 you don't use a container that preserves line breaks you will need to
 configure highlight.js to use the `<br>` tag:
 
-```javascript
+```js
 hljs.configure({useBR: true});
 
-$('div.code').each((i, block) => {
+document.querySelectorAll('div.code').forEach((block, i) => {
```
Comment (marcoscaceres) [inline README.md]:
```suggestion
document.querySelectorAll('div.code').forEach((block) => {
```

File: README.ru.md [hunk 3]
```diff
@@ -42,9 +42,9 @@ Highlight.js — это инструмент для подсветки синт
 Вот пример инициализации, эквивалентной вызову [`initHighlightingOnLoad`][1], но
 с использованием jQuery:
 
-```javascript
-$(document).ready(function() {
-  $('pre code').each(function(i, block) {
+```js
+document.addEventListener('DOMContentLoaded', (event) => {
+  document.querySelectorAll('pre code').forEach((block, i) => {
```
Comment (marcoscaceres) [inline README.ru.md]:
```suggestion
  document.querySelectorAll('pre code').forEach((block) => {
```

File: README.ru.md [hunk 4]
```diff
@@ -54,10 +54,10 @@ $(document).ready(function() {
 используете контейнер, не сохраняющий переводы строк, вам нужно сказать
 highlight.js использовать для них тег `<br>`:
 
-```javascript
+```js
 hljs.configure({useBR: true});
 
-$('div.code').each(function(i, block) {
+document.querySelectorAll('div.code').forEach((block, i) => {
```
Comment (marcoscaceres) [inline README.ru.md]:
```suggestion
document.querySelectorAll('div.code').forEach((block) => {
```

File: README.ru.md [hunk 5]
```diff
@@ -42,9 +42,9 @@ Highlight.js — это инструмент для подсветки синт
 Вот пример инициализации, эквивалентной вызову [`initHighlightingOnLoad`][1], но
 с использованием jQuery:
```
Comment (marcoscaceres) [inline README.ru.md]: Oh, if you speak Russian, could you remove jQuery from here too?

Comment (marcoscaceres) [reply README.ru.md]: (if not, I think it's probably safe to just delete ", но
 с использованием jQuery"

Comment (amilajack) [reply README.ru.md]: screen shot 2019-01-23 at 10 39 54 pm

Here's the translation. Sure, I'll remove it now

Comment (amilajack) [reply README.ru.md]: I'll change it to this:
screen shot 2019-01-23 at 10 39 54 pm

Comment (marcoscaceres) [reply README.ru.md]: Assuming without the jQuery part :)

Comment (amilajack) [reply README.ru.md]: 🤦‍♂️ wrong screenshot. Sorry

Comment (marcoscaceres) [reply README.ru.md]: hehe, I figured as much. No problem :)

**Comment by egor-rogov** [comment]
Do we still need README in Russian? It differs from English version in couple of places. If we do need, I can make necessary changes to it.

**Comment by marcoscaceres** [comment]
@egor-rogov, I don't have a strong opinion. The two sources I found is that English literacy is low in Russia (thus the Russian translation might be beneficial to some users):

* [One site claims 30% "speak"](https://www.rbth.com/news/2015/12/03/percentage-of-rusians-who-speak-english-doubles-to-30_547189) English... but it's unclear about reading literacy.  
* Mostly anecdotal: https://www.quora.com/Do-many-people-living-in-Russia-speak-English

The number of people reading the README in Russian is too low to show here ():
https://github.com/highlightjs/highlight.js/graphs/traffic

Which might be an indication that we can remove it. However, if you are ok to maintain it, please do. 

So, up to you if you want to invest time maintaining it.

**Pull request merged**